packages/material-ui/src/TableCell/TableCell.test.js

```diff
@@ -30,52 +35,65 @@ describe('<TableCell />', () => {
   describeConformance(<TableCell />, () => ({
     classes,
     inheritComponent: 'td',
-    mount: mountInTable,
+    mount: (node) => {
+      const wrapper = mount(
+        <table>
+          <tbody>
+            <tr>{node}</tr>
+          </tbody>
+        </table>,
+      );
+      return wrapper.find('tr').childAt(0);
+    },
     refInstanceof: window.HTMLTableCellElement,
     // invalid nesting otherwise
     testComponentPropWith: 'td',
   }));
 
   describe('prop: padding', () => {
     it('doesn not have a class for padding by default', () => {
-      const wrapper = mountInTable(<TableCell padding="default" />);
-      expect(findOutermostIntrinsic(wrapper).hasClass(classes.paddingDefault)).to.equal(false);
+      const { cell } = mountInTable(<TableCell padding="default" />);
+      expect(cell).to.not.have.class(classes.paddingDefault);
     });
 
     it('has a class when `none`', () => {
-      const wrapper = mountInTable(<TableCell className="woofTableCell" padding="none" />);
-      expect(findOutermostIntrinsic(wrapper).hasClass(classes.paddingNone)).to.equal(true);
+      const { cell } = mountInTable(<TableCell className="woofTableCell" padding="none" />);
```

Comment:
While we're at it, let's clean this up and remove unused attributes such as `className`